akka-docs/rst/java/code/docs/io/japi/IODocTest.java

```diff
@@ -90,6 +90,10 @@ public void onReceive(Object msg) throws Exception {
     
     final InetSocketAddress remote;
     final ActorRef listener;
+    
+    public static Props props(InetSocketAddress remote, ActorRef listener) {
+        return Props.create(TcpClientActor.class, remote, listener);
+    }
```

Comment:
Good catch, thanks! 
Would you mind doing the same for `Server` and using those `props` methods on lines 157, 159 so they don't show up as unused?